lua/nvim-treesitter/textobjects/select.lua

```diff
@@ -94,10 +94,18 @@ local val_or_return = function(val, opts)
   end
 end
 
-function M.select_textobject(query_string, query_group, keymap_mode)
+function M.select_textobject(query_string, query_group, keymap_mode, local_lookahead, local_lookbehind)
```

Comment:
I think the name of the args could be `lookahead` and `lookbehind`. If they are not defined just use the global. Or I would name it like `override_lookahead` etc.